src/index.js

```diff
@@ -38,6 +37,7 @@ const ow = require('openwhisk')
  * @property {ow.Rules} rules rules
  * @property {ow.Triggers} triggers triggers
  * @property {ow.Routes} routes routes
+ * @property {LogForwarding} routes logForwarding
```

Comment:
Should be `@property {LogForwarding} logForwarding Log Forwarding API` i suppose ? I will try and fix the duplicates issue later.

Reply:
Fixed